starkliup/starkliup

```diff
@@ -37,33 +37,53 @@ if test -t 1; then
 fi
 
 main() {
-    if [ -z "$1" ]; then
-        install
-        echo
-        completions
+    RELEASE_TAG=latest
 
-        echo
-        echo "Installation successfully completed."
-    else
-        case $1 in
+    while [ $# -gt 0 ]; do
+        case "$1" in
             -h|--help)
                 usage
                 exit 0
                 ;;
+
```

Comment:
Be consistent in spacing. Remove this line since we didn't have it between cases.